DPG/Tasks/AOTTrack/qaEventTrack.cxx

```diff
@@ -119,6 +120,7 @@ struct qaEventTrack {
       return;
     }
     const AxisSpec axisPt{binsPt, "#it{p}_{T} [GeV/c]"};
+    const AxisSpec axisInvPt{100, 0, 100, "#it{p}_{T} [GeV/c]"};
```

Comment:
Here the title should be changed, since you want 1/pt rather than pt

Reply:
Good catch